### DISCUSSION TITLE
Why is spreading array not behaving like str join when calling external command

### DISCUSSION THREAD

**Discussion by tdaron** [nushell/nushell#11704] [Q&A]
Hi ! 
Can someone explain me the difference between

```nu
zellij run ...$args -- bash -c $'($cmd)'
```

and
```nu
let arg_joined = ($args | str join " ")
zellij run $args_joined -- bash -c $'($cmd)'
```

The first option works, and the second one does not when args contains more than 1 element

**Comment by fdncred** [comment]
The second one makes a string of arguments which some programs can think is one big argument like `"this is one big argument"` vs `"this" "is" "many" "arguments"`. I think that's what is going on.
